config/300-awssqssource.yaml

```diff
@@ -69,55 +69,59 @@ spec:
                       For more details, please refer to the Amazon SQS Developer Guide at
                       https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/sqs-visibility-timeout.html.
                     type: string
-              credentials:
-                description: Credentials to interact with the Amazon SQS API.
-                  For more information about AWS security credentials, please refer to the AWS General Reference at
-                  https://docs.aws.amazon.com/general/latest/gr/aws-security-credentials.html.
+              auth:
+                description: Authentication method to interact with the Amazon SQS API.
                 type: object
                 properties:
-                  accessKeyID:
-                    description: Access key ID.
+                  credentials:
+                    description: Security credentials authentication. For more information about AWS security
+                      credentials, please refer to the AWS General Reference at
+                      https://docs.aws.amazon.com/general/latest/gr/aws-security-credentials.html.
```

Comment:
GitHub shows a bunch of changes in this file, but all I did (apart from adding `auth` and a description) was indent everything 2 spaces to the right 🤡